CONTRIBUTING.md

```diff
@@ -17,3 +17,32 @@ rover is an open source project developed by [Apollo GraphQL](https://apollograp
 - One approval from a member of the Apollo tooling team.
   - The team consists of @jbaxleyiii, @ashleygwilliams, @JakeDawkins, and @EverlastingBugstopper.
   - PRs with changes in `docs/` automatically add @StephenBarlow as a reviewer. PRs are not blocked on this approval.
+
+## Adding new commands/subcommands
+
+When adding a new command or a subcommand, you should make sure that the field in the new command struct is titled `command`. This allows our automated anonymous usage collection to work properly.
+
+Should you add arguments that are _not_ subcommands, the arguments you specify will automatically be reported to Apollo's Telemetry backend, unless you specify that an argument should be ignored with `#[serde(skip_serializing)]`. You should be careful to add the `skip_serializing` attribute to any fields that may contain data that may not be anonymous. This can be anything from a profile name to a path on the user's computer that could include their name.
+
+If you would like to verify that your new command works with the usage data capturing system, you can spin up a development server at `http://localhost:8787/telemetry` (or, if you'd like to use some other endpoint you may specify `APOLLO_TELEMETRY_URL`) and make sure the `command` section of the request body looks right. One example of proper usage collection is when the command `rover config profile show default --sensitive` is run, the request body should look something like this:
+
+```json
+{
+  "machine_id": "esd140f0-3f2d-12f5-a22a-d5231d7e1802",
+  "cli_version": "0.0.1",
+  "platform": {
+    "os": "linux",
+    "continuous_integration": null,
+  },
+  "command": {
+    "name": "config profile show",
+    "arguments": {
+      "sensitive": true
+    }
+  },
+  "session_id": "ead890f0-1f8a-43f2-a21e-a3721d7e1042",
+  "cwd_hash": "25b120fee6f41226b1db561ecb708ave187e6912fa33a2390f8e01ca94e89bb1",
+}
+```
+
+As you can see, `default` is left out here in `command.arguments` because it is a user-defined argument and could contain identifiable information, and has been labelled with the `#[serde(skip_serializing)] attribute.
```

Comment:
"`default` is left out here in" -> "`default` is omitted from"

"could contain" -> "might contain"

Missing a close backtick at the end here I think